deployment/aws/lambda/Dockerfile

```diff
@@ -5,7 +5,7 @@ FROM --platform=linux/amd64 public.ecr.aws/lambda/python:${PYTHON_VERSION}
 WORKDIR /tmp
 
 RUN pip install pip -U
-RUN pip install "titiler.application==0.6.0" "mangum>=0.10.0" -t /asset --no-binary pydantic
+RUN pip install "https://github.com/SenteraLLC/titiler/archive/path-prefix.tar.gz" "mangum>=0.10.0" -t /asset --no-binary pydantic
```

Comment:
This installs Sentera's forked TiTiler repo rather than the mainstream `titiler` project that pip would typically install.